packages/js-client/src/context.ts

```diff
@@ -0,0 +1,71 @@
+import { SimpleStorageContextParams } from './types';
+import { Context, ContextCore, ContextState } from '@aragon/sdk-client-common';
+
+// TODO
+// import from @aragon/sdk-client-common
```

Comment:
?

Reply:
I forgot to add this to the client-common package, so this is intended to be imported from there

Reply:
But what are we importing?